src/core_plugins/console/api_server/spec/generated/cluster.put_settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_cluster/settings": {
```

Comment:
There were auto-complete rules previously, are those no longer needed? https://github.com/elastic/kibana/pull/16550/files#diff-f4f93da88d545bb4eebb25af30d5a8beL36

Reply:
Nevermind, see it in the overrides.